src/main/java/axel/Axel.java

```diff
@@ -0,0 +1,80 @@
+package axel;
```

Comment:
I think its a good idea to have related classes in different packages e.g. a `task` package for task classes, a `command` package for commands, an `axelException` package for your `AxelException`s etc.